zerver/actions/streams.py

```diff
@@ -309,7 +311,9 @@ def do_unarchive_stream(stream: Stream, new_name: str, *, acting_user: UserProfi
             sender,
             stream,
             str(Realm.STREAM_EVENTS_NOTIFICATION_TOPIC_NAME),
-            _("Channel {channel_name} un-archived.").format(channel_name=new_name),
+            _("Channel {channel_link} has been unarchived.").format(
+                channel_link=f"#**{new_name}**",
```

Comment:
Can this be `_("Channel #**{channel_name}** has been unarchived.").format(channel_name=new_name)`. I guess this would mean a new translation would need to be added by translators but I guess this is the more correct format. We can wait for more feedback.